trajectory_tracker/src/trajectory_tracker.cpp

```diff
@@ -530,32 +509,31 @@ TrackerNode::TrackingResult TrackerNode::getTrackingResult(
   try
   {
     tf2::Stamped<tf2::Transform> odom_to_path;
-    tf2::fromMsg(
-        tfbuf_.lookupTransform(frame_odom_, path_header_.frame_id, ros::Time(0)), odom_to_path);
+    //    tf2::fromMsg(tfbuf_.lookupTransform(frame_odom_, path_header_.frame_id, tf2::TimePointZero), odom_to_path);
+    tf2::fromMsg(tfbuf_.lookupTransform(frame_odom_, path_header_.frame_id, rclcpp::Time(0)), odom_to_path);
+
     transform *= odom_to_path;
-    transform_delay = (ros::Time::now() - transform.stamp_).toSec();
+    transform_delay = tf2::durationToSec(tf2_ros::fromRclcpp(now()) - transform.stamp_);
```

Comment:
The conversion from `tf2::TimePoint` to `rclcpp::Time` should also be carefully performed.
The clock_type of the instance of `rclcpp::Time` class created by `tf2_ros::toRclcpp()` is RCL_SYSTEM_TIME, and an exception is thrown when compared to the instance created by `rclcpp::Node::now()` .
This problem is avoided by converting the return value of `rclcpp::Node::now()` to `tf2::TimePoint`.